### PR TITLE
Update NetworkUtils.java

### PR DESCRIPTION
**Pull request by davidgraeff**
Use --help on external tool to print the full usage message and then use a regex to find the expected string.
The THOMAS_HABERT_ARPING_WITHOUT_TIMEOUT variant is really just a fallback. Without
a timeout it may lock threads if the target device never comes online.

### CODE REVIEW

**Comment by kaikreuzer** [comment]
The builds do not like that change and you forgot the sign-off :-)

File: addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java [hunk 1]
```diff
@@ -198,11 +198,11 @@ public IpPingMethodEnum determinePingMethod() {
      * Return true if the external arp ping utility (arping) is available and executable on the given path.
      */
     public ArpPingUtilEnum determineNativeARPpingMethod(String arpToolPath) {
-        String result = ExecUtil.executeCommandLineAndWaitResponse(arpToolPath, 100);
+        String result = ExecUtil.executeCommandLineAndWaitResponse(arpToolPath+" --help", 100);
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java]: formatting looks a little of

**Comment by davidgraeff** [comment]
There is an escaping backslash missing

**Comment by davidgraeff** [comment]
Could you squash the changes and properly add the signed-off-by line while merging? I have no machine with a checkout here right now. Can do it on the weekend though.

**Comment by martinvw** [comment]
> Could you squash the changes and properly add the signed-off-by line while merging? I have no machine with a checkout here right now. Can do it on the weekend though.

If you can (just for the formality) explicitly state that you agree to the DCO for all your changes in this PR 😄

**Comment by davidgraeff** [comment]
All commits

Signed-off-by: David Graeff <david.graeff@web.de>

**Comment by martinvw** [comment]
> All commits
> 
> Signed-off-by: David Graeff [david.graeff@web.de](mailto:david.graeff@web.de)

Thanks and merged!

**Pull request merged**